integrations/data/TestBatch_StateDuration.0.brpl

```diff
@@ -0,0 +1,2 @@
+{"name":"cpu","tags":{"host":"serverA"},"points":[{"time":"1971-01-01T00:00:00Z","fields":{"value":97.1}},{"time":"1971-01-01T00:00:01Z","fields":{"value":96.6}},{"time":"1971-01-01T00:00:02Z","fields":{"value":83.6}},{"time":"1971-01-01T00:00:03Z","fields":{"value":99.1}}]}
+{"name":"cpu","tags":{"host":"serverB"},"points":[{"time":"1971-01-01T00:00:00Z","fields":{"value":47}},{"time":"1971-01-01T00:00:01Z","fields":{"value":95.1}},{"time":"1971-01-01T00:00:02Z","fields":{"value":null}},{"time":"1971-01-01T00:00:03Z","fields":{"value":96.1}}]}
```

Comment:
If it helps I updated the parser a while back to not care about newlines, so you can break this data up and format it like normal JSON. Not a needed change but useful to know.